docs/source/models/concept_models/vivarium_mncnh_portfolio/mncnh_interventions.rst

```diff
@@ -0,0 +1,130 @@
+.. mncnh_interventions:
+
+==================================
+CPAP for treating Preterm with RDS
+==================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. list-table:: Abbreviations
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+    - Note
+  * - CPAP
+    - Continuous positive airway pressure
+    - 
+  * - RDS
+    - respiratory distress syndrome
+    - 
+
+Intervention Overview
+-----------------------
+
+CPAP is used for the prevention and treatment of respiratory distress syndrome. A recent (2020)Cochrane Review found that CPAP was associated with lower risk of treatment failure (death or use of assisted ventilation) and lower overall mortality with moderate‐certainty evidence.
+
+This section describes how a CPAP intervention can be implemented and calibrated for the :ref:`MNCNH Portfolio model <2024_concept_model_vivarium_mncnh_portfolio>`.
+
+.. list-table:: Affected Outcomes
+  :widths: 15 15 15 15
+  :header-rows: 1
+
+  * - Outcome
+    - Effect
+    - Modeled?
+    - Note (ex: is this relationship direct or mediated?)
+  * - Preterm with RDS Mortality Probability
+    - Decrease multiplicatively
+    - Yes
+    - 
+
+Baseline Coverage Data
+++++++++++++++++++++++++
+
+The :ref:`MNCNH Portfolio model <2024_concept_model_vivarium_mncnh_portfolio>` includes data on current coverage levels for the specific locations we are focused on modeling.  We might wish to enhance this by predicting coverage in additional locations, in correlation with other key intervention technologies, based on sources such as existing SPA and SARA data.
```

Comment:
What are SPA and SARA?

Reply:
Yes, what are those? Also, this sentence is rather cryptic to me -- why would we want to expand to other locations besides the ones we're modeling, and what other intervention technologies are you talking about?